advisor/src/funTest/kotlin/OsvFunTest.kt

```diff
@@ -77,7 +78,7 @@ class OsvFunTest : StringSpec({
 })
 
 private fun identifierToPackage(id: String): Package =
-    Package.EMPTY.copy(id = Identifier(id))
+    Identifier(id).let { Package.EMPTY.copy(id = it, purl = it.toPurl()) }
```

Comment:
Would swapping the order of this and the following commit lead to the situation that all tests pass of all of the three commits of this PR? If so, would it make sense to change the order of the top-most two commits?

Reply:
Yes, that would be the case, but I deliberately did not do so as I found it strange to commit a fixup for a commit that I reverted before. That could raise the question whether the fixup should have also been reverted.